### ISSUE TITLE
Fails to download

### ISSUE THREAD

**Issue by alexkreidler** [mbalabash/find-chrome-bin#1]
```
$ ts-node --esm ./src/utils/puppeteer.ts 
/bin/sh: 1: undefined: not found
ERROR: Failed to download Chromium!
TypeError: puppeteer.BrowserFetcher is not a constructor
    at downloadChromium (/home/alex/toky/backend/v4/node_modules/.pnpm/find-chrome-bin@1.0.5/node_modules/find-chrome-bin/src/chrome/index.js:52:26)
    at findChrome (/home/alex/toky/backend/v4/node_modules/.pnpm/find-chrome-bin@1.0.5/node_modules/find-chrome-bin/index.js:27:32)
    at getBrowser (file:///home/alex/toky/backend/v4/src/utils/puppeteer.ts:17:26)
    at file:///home/alex/toky/backend/v4/src/utils/puppeteer.ts:119:1
    at ModuleJob.run (node:internal/modules/esm/module_job:193:25)
    at async Promise.all (index 0)
    at async ESMLoader.import (node:internal/modules/esm/loader:530:24)
    at async loadESM (node:internal/process/esm_loader:91:5)
    at async handleMainPromise (node:internal/modules/run_main:65:12)
 ```

```
    "puppeteer": "^20.9.0",
    "puppeteer-core": "^20.9.0",
    "find-chrome-bin": "^1.0.5",
```

**Comment by NERLOE** [comment]
I have the exact same issue

**Comment by mbalabash** [comment]
Should be fixed in `find-chrome-bin@2.0.0`
